lib/exception_handling/methods.rb

```diff
@@ -16,7 +16,9 @@ def log_error_rack(exception_or_string, exception_context = '', rack_filter = ''
     end
 
     def log_warning(message)
-      log_error(Warning.new(message))
+      warning = Warning.new(message)
+      warning.set_backtrace([])
+      log_error(warning)
```

Comment:
Both of these `log_warning` methods need to be tested, but one isn't, right?  I'm guessing you didn't TDD that one?